### PR TITLE
[EC-226] Add FIMS networking module

### PR DESCRIPTION
**Pull request by Krusty93**
### List of changes

Add networking module:
- subnet
- nat association

### Motivation and context

Bringing here FIMS resources

### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [ ] DEV
- [ ] UAT
- [ ] PROD

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [X] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [X] No

### Other information

---

### CODE REVIEW

File: infra/prod/westeurope/networking.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module "networking" {
+  source = "../_modules/networking"
+
+  location = local.location
+  project  = local.project
+
+  subnet_fims_cidr = "10.0.18.0/26"
```
Comment (gunzip) [inline infra/prod/westeurope/networking.tf:7]: how do you choose this cidr? can you add a comment?

Comment (Krusty93) [reply infra/prod/westeurope/networking.tf:7]: The subnet already exixsts, I have actually imported it from `io-infra`.
I believe it is a simple progressive enumeration

Comment (gunzip) [reply infra/prod/westeurope/networking.tf:7]: I'd add a reference to that one anyway.

Comment (Krusty93) [reply infra/prod/westeurope/networking.tf:7]: here you go #12

**Pull request merged**